plugin/src/cli_utils/localpvhostpath/mod.rs

```diff
@@ -0,0 +1,76 @@
+use crate::cli_utils::CliArgs;
+use clap::Parser;
+use plugin::ExecuteOperation;
+
+/// LocalPV Hostpath operations.
+#[derive(Parser, Debug)]
+pub enum Operations {
+    /// Gets localpv-hostpath resources.
+    #[clap(subcommand)]
+    Get(HosthpathGet),
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(hostpathget) => {
+                hostpathget.execute(cli_args).await?;
+            }
+        }
+        Ok(())
+    }
+}
+
+#[derive(clap::Subcommand, Debug)]
+pub enum HosthpathGet {
+    /// Gets a specific localpv-hostpath volume.
+    Volume(GetVolumeArg),
+    /// Lists all localpv-hostpath volumes.
+    Volumes(GetVolumesArg),
+}
+
+/// Argument used when getting a hostpath volume.
+#[derive(Debug, Clone, clap::Args)]
+pub struct GetVolumeArg {
+    /// Volume id of the volume.
+    volume: String,
```

Comment:
are all these missing #[arg(long, .... ] ? or is it not required?

Reply:
its not required i think. This just represents the arg object. Not a command.

Reply:
Are you sure? Does it show on --help?

Reply:
Yes, see..
```
[nix-shell:~/mayarepo/openebs-new/openebs]# ./target/debug/kubectl-openebs localpv-lvm get volumes --help
Lists all localpv-lvm volumes

Usage: kubectl-openebs localpv-lvm get volumes [OPTIONS] [NODE_ID]
```

Reply:
Right, it defaults to a positional argument, rather than a flag.
You should check in every case if that's what you want.

Reply:
Yes, We have 1 argument in all case. Its optiaonal or mandatory. For eg:

Since user wants to get a specific volume , volume id is required
```
./target/debug/kubectl-openebs localpv-lvm get volume
error: the following required arguments were not provided:
  <VOLUME>
  
```

In volumes command node_id is not required. User can specify it.

```

[nix-shell:~/mayarepo/openebs-new/openebs]# ./target/debug/kubectl-openebs localpv-lvm get volumes
 NAME  NODE  STATUS  CAPACITY  VOLGROUP  PVC-NAME  SC-NAME 

[nix-shell:~/mayarepo/openebs-new/openebs]# ./target/debug/kubectl-openebs localpv-lvm get volumes --help
Lists all localpv-lvm volumes

Usage: kubectl-openebs localpv-lvm get volumes [OPTIONS] [NODE_ID]

Arguments:
  [NODE_ID]  Lists lvm volumes from a specific node if set
```

Reply:
I dont see any usescases for multiple (required) params as of now. @Abhinandan-Purkait , @niladrih thought?